dist/index.html

```diff
@@ -13,38 +13,19 @@ <h1>Leaderboard</h1>
         <div class="recent-scores">
           <h3>Recent Scores</h3>
 
-          <input id="refresh-btn" type="button" value="Refresh"/>
-
+          <input id="refresh-btn" type="button" value="Refresh" />
         </div>
-        <ul id="score-list">
-          <li class="list">Name: 120</li>
-          <li class="list">Name: 20</li>
-          <li class="list">Name: 50</li>
-          <li class="list">Name: 78</li>
-          <li class="list">Name: 125</li>
-          <li class="list">Name: 77</li>
-          <li class="list">Name: 42</li>
-          <li class="list">Name: 180</li>
-        </ul>
+        <ul class="score-list"></ul>
       </div>
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thanks so much for this sugestion.